### PR TITLE
fix(): add exportAs to missing components

### LINKED ISSUE #7361: Create exportAs aliasing for all Components/Directives to allow for template referencing
This issue is meant to track which `@Component`s and `@Directive`s include `exportAs`aliases:

- [x] MatAutocomplete
- [x] MatButton
- [x] MatAnchor
- [x] MatButtonToggleGroup
- [x] MatButtonToggle
- [x] MatCard
- [x] MatChip
- [x] MatChipList
- [x] MatDatepicker
- [x] MatDatepickerInput
- [x] MatAccordion
- [x] MatExpansionPanel
- [x] MatFormField
- [x] MatGridList
- [x] MatGridTile
- [x] MatIcon
- [x] MatInput
- [x] MatTextareaAutosize
- [x] MatList
- [x] MatListItem
- [x] MatMenuItem
- [x] MatMenu
- [x] MatMenuTrigger
- [x] MatSelect
- [x] MatPaginator
- [x] MatProgressBar
- [x] MatProgressSpinner
- [x] MatRadioGroup
- [x] MatRadioButton
- [x] MatHorizontalStepper
- [x] MatVerticalStepper
- [x] MatStep
- [x] MatTable
- [x] MatRow
- [x] MatHeaderRow
- [x] MatTab
- [x] MatTabGroup
- [x] MatToolbar
- [x] MatToolbarRow
- [x] MatTooltip

### PR DESCRIPTION
**Pull request by DevVersion**
* Adds the `exportAs` option to the missing components inside of Angular Material

@jelbourn @josephperrott A few components/directives have been skipped because they were not added to the list in #7361. e.g. the `MdStepper` or basically all the CDK directives of the table/stepper.

**Note**: Adding the `exportAs` to the `validate-decorators` TSLint rule is not a solution right now. There are too many components/directives that shouldn't specify an `exportAs`. @crisbeto we can chat about this.

Fixes #7361

### CODE REVIEW

**Comment by DevVersion** [comment]
@jelbourn @josephperrott Updated the PR and included some more components/directives

File: src/cdk/portal/portal-directives.ts [hunk 1]
```diff
@@ -49,6 +49,7 @@ export class TemplatePortalDirective extends TemplatePortal<any> {
  */
 @Directive({
   selector: '[cdkPortalHost], [portalHost]',
+  exportAs: 'cdkPortalHost, portalHost',
```
Comment (josephperrott) [inline src/cdk/portal/portal-directives.ts]: I believe the intention is to remove `[portalHost]` eventually right?  If that is the case, should we not export as `portalHost`?

Comment (DevVersion) [reply src/cdk/portal/portal-directives.ts]: Makes sense. Updated it

File: src/lib/form-field/form-field.ts [hunk 2]
```diff
@@ -53,6 +53,7 @@ let nextUniqueId = 0;
   moduleId: module.id,
   // TODO(mmalerba): the input-container selectors and classes are deprecated and will be removed.
   selector: 'mat-input-container, mat-form-field',
+  exportAs: 'matFormField, matInputContainer',
```
Comment (josephperrott) [inline src/lib/form-field/form-field.ts]: Since `<mat-input-container>` is being deprecated, should we not include it in `exportAs`

Comment (DevVersion) [reply src/lib/form-field/form-field.ts]: Makes sense. Removed it

**Pull request merged**